build.gradle

```diff
@@ -226,6 +226,7 @@ task nouitest(type: Test) {
 
 task uitest(type: Test) {
     group = "Verification"
+    maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```

Comment:
It's always faster to run the UI tests in parallel.  Not so much the other kinds though.